beefy-gadget/src/lib.rs

```diff
@@ -123,7 +121,7 @@ pub async fn start_beefy_gadget<B, P, BE, C, N, SO>(
 	P::Public: AppPublic + Codec,
 	P::Signature: Clone + Codec + Debug + PartialEq + TryFrom<Vec<u8>>,
```

Comment:
Both `P::Public` and `P::Signature` bounds are redundant as well now, no?

Reply:
Don't thing so, `P::Public` is used as a bound for `BeefyApi` and `P::Signature` is a bound on `BeefyWorker`. I think, might could do with one less bound on `BeefyWorker`. Need to look into that later.

Going to merge the PR for now